gnucash/import-export/import-backend.c

```diff
@@ -1091,10 +1091,18 @@ static gint check_trans_online_id(Transaction *trans1, void *user_data)
     }
 }
 
+typedef struct _collect_trans_online_id_data
```

Comment:
No need to name the struct when you typedef the definition, you can just say
```
typedef struct
{
/* ... */
} collect_trans_online_id_data;
```

Reply:
yep I'm aware of that. Only replicating what I see in the existing code so I'm changing the code style.